src/Hotel.js

```diff
@@ -20,12 +20,32 @@ class Hotel {
     })
   }
 
+  listBookedRoomNumberByDate(date) {
+    const bookedRoomsOnDate = this.bookings.filter(booking => {
+      return booking.date === date;
+    })
+    return bookedRoomsOnDate.map(booked => {
+      return booked.roomNumber
+    })
+  }
+
+  listVacantRoomsByDate(date) {
+    const bookedRoomNumbers = this.listBookedRoomNumberByDate(date)
+    const availableRoomsOnDate = this.rooms.reduce((available, room) => {
+      !bookedRoomNumbers.includes(room.number) ? available.push(room) : null;
```

Comment:
nice ternary but if you did an if statement, you wouldn't need another condition for null